app/src/main/java/com/tohami/newsapi/ui/news/ArticleFragment.java

```diff
@@ -0,0 +1,57 @@
+package com.tohami.newsapi.ui.news;
+
+import android.net.Uri;
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import androidx.annotation.Nullable;
+import androidx.browser.customtabs.CustomTabsIntent;
+import androidx.databinding.DataBindingUtil;
+import androidx.lifecycle.ViewModelProviders;
+
+import com.tohami.newsapi.R;
+import com.tohami.newsapi.base.BaseFragment;
+import com.tohami.newsapi.databinding.FragmentArticleBinding;
+import com.tohami.newsapi.di.Injection;
+
+public class ArticleFragment extends BaseFragment {
+
+    private FragmentArticleBinding binding;
+
+
+    @Override
+    public void onActivityCreated(@Nullable Bundle savedInstanceState) {
+        super.onActivityCreated(savedInstanceState);
+        NewsViewModel mViewModel = ViewModelProviders.of(getActivity(), Injection.provideViewModelFactory(getContext()))
```

Comment:
Details should not use the same view model as the listing

Reply:
done